### PR TITLE
bugfix: locked down boulder commit used in build

### PR DESCRIPTION
**Pull request by m59peacemaker**
Recommending this build be tagged 1.0.1 on DockerHub, or delete 1.0.0 and replace it with this, since 1.0.0 is not functional.

### CODE REVIEW

**Comment by jderusse** [comment]
Whath wrong with master?

File: Dockerfile [hunk 1]
```diff
@@ -25,7 +25,11 @@ ENV GO15VENDOREXPERIMENT 1
 WORKDIR /go/src/github.com/letsencrypt/boulder
 
 RUN mkdir -p /go/src/github.com/letsencrypt \
- && git clone --depth 1 --branch master https://github.com/letsencrypt/boulder.git /go/src/github.com/letsencrypt/boulder
+ && git clone https://github.com/letsencrypt/boulder.git /go/src/github.com/letsencrypt/boulder \
+ && cd /go/src/github.com/letsencrypt/boulder \
+ && git reset --hard e45cd826f7cc207a063df6b0130c52395d3e481b \
+ && rm -rf .git
+ # 5/16/2016
```
Comment (jderusse) [inline Dockerfile:32]: ?

**Comment by m59peacemaker** [comment]
I followed up about it here https://github.com/acmephp/testing-ca/issues/6#issuecomment-237445532

You can't guarantee that every commit to a github repo is stable enough to use. Your 1.0.0 image is different than the previous build from 4/22, since it used a different commit. That commit isn't producing a working build with your Dockerfile. The server fails to start. You should always lock things down to specific versions / commits / tags for this reason.

**Comment by jderusse** [comment]
Ok, I see the problem :/

Is there a way to se  the July's tag and fix the image ?

**Comment by m59peacemaker** [comment]
Probably, but I don't know much about boulder. Your image is the only one I could get working at all. boulder now has it's own docker setup in the repo that is supposed to start with `docker-compose up`, but it's broken, too. https://github.com/letsencrypt/boulder#quickstart

**Comment by m59peacemaker** [comment]
Using the July release is ideal, but the older commits are working well.

**Comment by jderusse** [comment]
You're right. This acme-testing-ca was using an old commit for months and
works well.

Let's do merge this PR to expose a functional image and create a new PR
with an earlier version of boulder.

@tgalopin what's your insight?

Le jeu. 4 août 2016 08:23, Johnny Hauser notifications@github.com a
écrit :

**Comment by jderusse** [comment]
Did you test your image before submitting this PR ?

**Comment by m59peacemaker** [comment]
I use your server image to test my client image https://github.com/m59peacemaker/docker-autossl

The tests passed on the old build from Apr. 22 and they also pass with my PR with the commit from 5/16. Here's where I call the `certbot` command if it helps get a feel for the kinds of options I'm testing. https://github.com/m59peacemaker/docker-autossl/blob/master/lib/cert-command.js

**Comment by tgalopin** [comment]
Could you try using commit a8ac802f9963b1ab9e42fe67438f941a6c3dd8fb (tagged as release-2016-07-14)? Using a real release seems better IMO.

I'll try too on my local machine.

**Comment by tgalopin** [comment]
Okay, it does not seem to work with this release. It works with fd374224682dae639829e4b0eabcf2e12e381755 (released as hotfixes/2016-06-30) for me, can you confirm and use this commit in your PR?

EDIT: I was too fast, it does not seem to work either. I merge this PR :) .

**Comment by tgalopin** [comment]
Thanks @m59peacemaker !

**Pull request merged**